src/output/default.rs

```diff
@@ -443,11 +444,13 @@ impl OutputVisitor for DefaultOutput {
             TestResult::Fail(panic_info, captured_stdout, captured_stderr) => {
                 self.writeln_cmt(&format!("✘ {}", msg), cmt, indent, Color::Red, false);
                 self.print_step_extras(step);
+                let tw = textwrap::termwidth();
+                let c_count = panic_info.location.chars().count();
                 self.writeln_cmt(
                     &format!(
                         "{:—<1$}",
                         "! Step failed: ",
-                        textwrap::termwidth() - panic_info.location.chars().count() - 7
+                        if c_count + 7 < tw { tw - c_count - 7 } else { 0 },
```

Comment:
Use `std::cmp::max`, it will read more clearly (ie, `max(0, ...)` will ensure a minimum of zero)

Reply:
`saturating_sub` also seems appropriate?